tests/Action/AppendFormFieldElementActionTest.php

```diff
@@ -118,9 +110,9 @@ public function testAppendFormFieldElementAction(): void
 
     private function configureFormRenderer()
     {
-        $runtime = $this->prophesize(FormRenderer::class);
+        $runtime = $this->createStub(FormRenderer::class);
```

Comment:
```suggestion
        $runtime = $this->createMock(FormRenderer::class);
```
There is this call in `testAppendFormFieldElementAction`:
```php
$renderer->expects($this->once())->method('setTheme')->with($formView);
```